raster/src/test/scala/geotrellis/raster/ArrayTileSpec.scala

```diff
@@ -186,18 +188,19 @@ class ArrayTileSpec extends FunSpec
 
       withClue("IntArrayTile with NoData") {
         val at = injectNoData(4)(createConsecutiveTile(10))
-          .convert(DoubleUserDefinedNoDataCellType(98))
+          .convert(IntUserDefinedNoDataCellType(98))
```

Comment:
I think this is a copy/paste mistake... we should be testing integral tiles here.